mods/ts/rules/nod-vehicles.yaml

```diff
@@ -73,6 +73,13 @@ TTNK:
 	Tooltip:
 		Name: Tick Tank
 		Description: Nod's main battle tank.\nCan deploy to gain extra protection.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
+		UpgradeTypes: deployed
+		UpgradeMaxEnabledLevel: 0
+	Tooltip@DEPLOYED:
+		Name: Deployed Tick Tank
```

Comment:
Deployed Tick Tank is what vanilla TS uses. But i think Tick Tank (deployed)  is better too.